layouts/partials/fragments/item.html

```diff
@@ -178,10 +178,10 @@ <h5>
                   <thead>
                     <tr>
                       {{ $.Scratch.Set "col" 0 }}
-                      {{ range $data.header.values }}
+                      {{ range $item.header.values }}
                         <th class="
-                          {{- if $data.unimportant -}}
-                            {{- if index $data.unimportant ($.Scratch.Get "col") -}}
+                          {{- if $item.unimportant -}}
+                            {{- if index $item.unimportant ($.Scratch.Get "col") -}}
```

Comment:
Let's remove the code related to unimportant. This should solve the current table issue.

Reply:
Was that issue with Scratch resolved? Thought it was pushed back and therefore the code would be removed and an issue created in the long-run

Reply:
I thought I had removed it already. Is this change supposed to reflect in all fragments?

Reply:
Not sure where you removed it, but I would assume, that it would also break here as $.Scratch.Get wouldn't be accessible afaik. Therefore removing makes sense.

Reply:
Alright, I completely removed it.